xapian-letor/api/letor_internal.cc

```diff
@@ -180,6 +180,12 @@ static void read_problem(const char *filename) {
 	}
 	++elements;
 	++prob.l;
+vector<Xapian::RankList>
```

Comment:
It indeed is funky. The reason I separated it from the commit that tidies up the code was because of the whitespace issues you pointed out in my mail on the mailing list. (https://github.com/ayshtmr/xapian/commit/2abaf6b8d64b1624699c3f490b6dc441e1c6cdf3)
It was probably an issue with my editor configuration, but I couldn't understand what was happening even after indentation setting were set fine. Hence, I separated the commits so that it looks fine in the overall diffs. I know it's not a good practise, though.